test/Test/Data/Array.purs

```diff
@@ -363,10 +363,10 @@ testArray = do
   assert $ A.groupBy (\_ _ -> true) [1, 2, 3] == [nea [1, 2, 3]]
 
   log "groupAllBy should group equal elements into arrays based on an equivalence relation"
```

Comment:
What about “based on the result of a comparison function”, as in the test for sortBy?